server/service/media/endpoints.js

```diff
@@ -0,0 +1,35 @@
+const { asyncMiddleware } = require('../../util/api');
+const multer = require('multer');
+const { uploadToS3 } = require('./s3');
+
+async function uploadAudio(req, res) {
+    const storage = multer.memoryStorage();
+    const upload = multer({ storage: storage });
```

Comment:
Nit: You can use object shorthand here like `const upload = multer({ storage });`

Reply:
Though I'm seeing this is the same as the Multer documentation example. It might be worth seeing if you can clean it up a bit, but no big deal if not.